browser/ui/webui/brave_webui_source.cc

```diff
@@ -223,6 +223,8 @@ void CustomizeWebUIHTMLSource(const std::string &name,
         { "adsDisabledTextTwo",  IDS_BRAVE_REWARDS_LOCAL_ADS_DISABLED_TEXT_TWO },                // NOLINT
         { "adsNotificationsReceived",  IDS_BRAVE_REWARDS_LOCAL_ADS_NOTIFICATIONS_RECEIVED },     // NOLINT
         { "adsNotSupported", IDS_BRAVE_REWARDS_LOCAL_ADS_NOT_SUPPORTED },
+        { "adsNotSupportedRegion", IDS_BRAVE_REWARDS_LOCAL_ADS_NOT_SUPPORTED_REGION },           // NOLINT
```

Comment:
where is this used?

Reply:
`components/resources/brave_components_strings.grd:      <message name="IDS_BRAVE_REWARDS_LOCAL_ADS_NOT_SUPPORTED_REGION" desc="">Sorry! Ads are not yet available in your region.</message>`

Reply:
@ryanml you added it a while ago. Is it used today?

Reply:
@ryanml @NejcZdovc as far as I understand we need it. Can we resolve this conversation?

Reply:
Removed